pyproject.toml

```
@@ -63,7 +63,10 @@ Distribution = "https://pypi.org/project/towncrier"
 exclude = [
     "admin",
     "bin",
-    "docs",
+    ".github",
+    ".git-blame-ignore-revs",
+    ".pre-commit-config.yaml,
```

fix toml list syntax

```suggestion
    ".pre-commit-config.yaml",
```